pom.xml

```diff
@@ -773,25 +771,42 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>2.22.2</version>
+					<version>3.0.0-M5</version>
 					<configuration>
 						<encoding>UTF-8</encoding>
 						<argLine>-Xmx1G</argLine>
+						<skipTests>${skipUnitTests}</skipTests>
```

Comment:
Btw. This make it so that the regular `-DskipTests` doesn't work. I wonder if there is some way to set `skipUnitTests` to true if `skipTests` is true. For the sanity of other developers I think it's best that the default maven properties still work, so they don't need to hunt in our documentation to figure out they should use the quick profile.

Reply:
Good point, I'll take a look at this.

Reply:
This should be fixed now btw.